h2olog

```diff
@@ -681,7 +688,39 @@ int trace_h2o__send_response_header(struct pt_regs *ctx) {
 
     return 0;
 }
-#endif /* ENABLE_SEND_RESPONSE_HEADER */
+
```

Comment:
Similarly, for consistency's sake, you might want to delete this empty line.

Reply:
done! bbc3cc817cb98c5eaed720bfeced14af235fdec6